icgc-argo-workflows/data-processing-utility-tools
# Payload gen seq experiment@0.6.0.1

- Added support for `core` and `conditional`.
  - https://github.com/icgc-argo-workflows/data-processing-utility-tools/blob/baf295868c44739b367874251068b7511fee4db7/payload-gen-seq-experiment/main.py#L40-61
  - Should fail if core is not detected
- Fail if `library_strandedness` is not present if `experiment_type==RNA-seq`
  - https://github.com/icgc-argo-workflows/data-processing-utility-tools/blob/baf295868c44739b367874251068b7511fee4db7/payload-gen-seq-experiment/main.py#L328-L329
- Validation check against schema
  - https://github.com/icgc-argo-workflows/data-processing-utility-tools/blob/baf295868c44739b367874251068b7511fee4db7/payload-gen-seq-experiment/main.py#L179-246
  - Check wrong enum value in `platform`
  - Check wrong regex in `submitter_sequencing_experiment_id`
  - incorrect min max number in `rin`
